conjur/login/login_controller.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginController module
+
+This module is the controller that facilitates all login actions
+required to successfully configure the netrc
+"""
+
+# Builtins
+import getpass
+import logging
+import sys
+
+from conjur.init import ConjurrcData
+from conjur.utils import Utils
+import conjur.constants
+
+class LoginController:
+    """
+    LoginController
+
+    This class represents the Presentation Layer for the LOGIN command
+    """
+    password = None
+    credential_data = None
+    login_logic = None
+    ssl_verify = None
+
+    def __init__(self, ssl_verify, password, credential_data, login_logic):
+        self.ssl_verify = ssl_verify
+        if self.ssl_verify is False:
+            Utils.insecure_message()
+
+        self.password = password
+        self.credential_data = credential_data
+        self.login_logic = login_logic
+
+    def load(self):
+        """
+        Method that facilitates all method calls in this class
+        """
+        self.get_username()
+        self.password = self.get_password()
+        loaded_conjurrc = self.load_conjurrc_data()
```

Comment:
No need to call it `loaded`.
`conjurrc` is enough. (which is somewhat a weird name, as it says nothing about its content, but whatever...)

Reply:
what about init_details? or config?

Reply:
I'm all for removing the `conjurrc` concept (together with the `netrc`).
maybe `server_config`?